LoRaEngine/LoraKeysManagerFacade/CreateEdgeDevice.cs

```diff
@@ -34,98 +34,96 @@ public static async System.Threading.Tasks.Task<HttpResponseMessage> RunAsync([H
+            twin.Properties.Desired = new TwinCollection(@"{FacadeServerUrl:'" + string.Format("https://{0}.azurewebsites.net/api/", GetEnvironmentVariable("FACADE_HOST_NAME")) + "',FacadeAuthCode: " +
                 "'" + facadeKey + "'}");
             var remoteTwin = await manager.GetTwinAsync(deviceName);
 
             await manager.UpdateTwinAsync(deviceName, "LoRaWanNetworkSrvModule", twin, remoteTwin.ETag);
 
             bool deployEndDevice = false;
-            Boolean.TryParse(Environment.GetEnvironmentVariable("DEPLOY_DEVICE"),out deployEndDevice);
+            bool.TryParse(Environment.GetEnvironmentVariable("DEPLOY_DEVICE"), out deployEndDevice);
 
-            //This section will get deployed ONLY if the user selected the "deploy end device" options.
-            //Information in this if clause, is for demo purpose only and should not be used for productive workloads.
+            // This section will get deployed ONLY if the user selected the "deploy end device" options.
+            // Information in this if clause, is for demo purpose only and should not be used for productive workloads.
             if (deployEndDevice)
             {
-                string OTAAdeviceId = "47AAC86800430028";
-                Device OTAADevice = new Device(OTAAdeviceId);
-                await manager.AddDeviceAsync(OTAADevice);
-                Twin OTAAendTwin = new Twin();
-                OTAAendTwin.Properties.Desired = new TwinCollection(@"{AppEUI:'BE7A0000000014E2',AppKey:'8AFE71A145B253E49C3031AD068277A1',GatewayID:''," +
+                string oTAAdeviceId = "47AAC86800430028";
+                Device oTAADevice = new Device(oTAAdeviceId);
+                await manager.AddDeviceAsync(oTAADevice);
+                Twin oTAAendTwin = new Twin();
+                oTAAendTwin.Properties.Desired = new TwinCollection(@"{AppEUI:'BE7A0000000014E2',AppKey:'8AFE71A145B253E49C3031AD068277A1',GatewayID:''," +
                 "SensorDecoder:'DecoderValueSensor'}");
-                var OTAARemoteTwin = await manager.GetTwinAsync(OTAAdeviceId);
-                await manager.UpdateTwinAsync(OTAAdeviceId, OTAAendTwin, OTAARemoteTwin.ETag);
-                string ABPdeviceId = "46AAC86800430028";
-                Device ABPDevice = new Device(ABPdeviceId);
-                await manager.AddDeviceAsync(ABPDevice);
-                Twin ABPTwin = new Twin();
-                ABPTwin.Properties.Desired = new TwinCollection(@"{AppSKey:'2B7E151628AED2A6ABF7158809CF4F3C',NwkSKey:'3B7E151628AED2A6ABF7158809CF4F3C',GatewayID:''," +
+                var oTAARemoteTwin = await manager.GetTwinAsync(oTAAdeviceId);
+                await manager.UpdateTwinAsync(oTAAdeviceId, oTAAendTwin, oTAARemoteTwin.ETag);
+                string aBPdeviceId = "46AAC86800430028";
+                Device aBPDevice = new Device(aBPdeviceId);
+                await manager.AddDeviceAsync(aBPDevice);
+                Twin aBPTwin = new Twin();

```

Comment:
nit: abpDeviceId is more readable that aBPdeviceId
Same applies to abpTwin instead of aBPTwin